apps/myaccount/src/store/actions/authenticate.ts

```diff
@@ -234,31 +237,49 @@ export const getProfileInformation = (updateProfileCompletion = false) => (dispa
 };
 
 export const initializeAuthentication = () =>(dispatch)=> {
+
     const auth = IdentityClient.getInstance();
 
-    auth.on(Hooks.HttpRequestError, onHttpRequestError);
-    auth.on(Hooks.HttpRequestFinish, onHttpRequestFinish);
-    auth.on(Hooks.HttpRequestStart, onHttpRequestStart);
-    auth.on(Hooks.HttpRequestSuccess, onHttpRequestSuccess);
+    const responseModeFallback: ResponseModeTypes = process.env.NODE_ENV === "production"
+        ? "form_post"
+        : "query";
+
+    const storageFallback: Storage = new UAParser().getBrowser().name === "IE"
+        ? Storage.SessionStorage
+        : Storage.WebWorker;
 
     const initialize = (response?: any): void => {
         auth.initialize({
             authorizationCode: response?.data?.authCode,
             baseUrls: [window["AppUtils"].getConfig().serverOrigin],
             clientHost: window["AppUtils"].getConfig().clientOriginWithTenant,
             clientID: window["AppUtils"].getConfig().clientID,
-            enablePKCE: true,
-            responseMode: process.env.NODE_ENV === "production" ? "form_post" : null,
-            scope: [TokenConstants.SYSTEM_SCOPE],
-            serverOrigin: window["AppUtils"].getConfig().serverOriginWithTenant,
+            enablePKCE: window["AppUtils"].getConfig().idpConfigs?.enablePKCE
+                ?? true,
+            endpoints: {
+                authorize: window["AppUtils"].getConfig().idpConfigs?.authorizeEndpointURL,
+                jwks: window["AppUtils"].getConfig().idpConfigs?.jwksEndpointURL,
+                logout: window["AppUtils"].getConfig().idpConfigs?.logoutEndpointURL,
+                oidcSessionIFrame: window["AppUtils"].getConfig().idpConfigs?.oidcSessionIFrameEndpointURL,
+                revoke: window["AppUtils"].getConfig().idpConfigs?.tokenRevocationEndpointURL,
+                token: window["AppUtils"].getConfig().idpConfigs?.tokenEndpointURL,
+                wellKnown: window["AppUtils"].getConfig().idpConfigs?.wellKnownEndpointURL
+            },
+            responseMode: window["AppUtils"].getConfig().idpConfigs?.responseMode
+                ?? responseModeFallback,
+            scope: window["AppUtils"].getConfig().idpConfigs?.scope
+                ?? [ TokenConstants.SYSTEM_SCOPE ],
+            serverOrigin: window["AppUtils"].getConfig().idpConfigs?.serverOrigin
+                ?? window["AppUtils"].getConfig().idpConfigs.serverOrigin,
             sessionState: response?.data?.sessionState,
             signInRedirectURL: window["AppUtils"].getConfig().loginCallbackURL,
             signOutRedirectURL: window["AppUtils"].getConfig().loginCallbackURL,
-            storage: new UAParser().getBrowser().name === "IE" ? Storage.SessionStorage : Storage.WebWorker
+            storage: window["AppUtils"].getConfig().idpConfigs?.storage
```

Comment:
Since our web worker implementation doesn't work properly in IE, we need to use session storage in IE. Are we making this conditional choice elsewhere?

Reply:
`storageFallback` has that condition. It should be enough right?